src/dapps/ManageENS/ManageENS.vue

```diff
@@ -252,12 +252,19 @@ export default {
         ResolverAbi,
         publicResolverAddress
       );
+      const getDecodedAddress = _coinItem => {
+        let decodedAddress = '0x';
```

Comment:
Should be either `Buffer.from("")` or `new Buffer(0)` to set empty string. cc @Arachnid

Reply:
Abi encoder converts '0x' to an empty buffer.